bindings/r/tests/testthat/_snaps/delimiter.md

```diff
@@ -53,8 +53,8 @@
       
       S-Expression
       (braced_expression [(2, 0), (3, 0)]
-        "{" [(2, 0), (2, 1)]
-        (ERROR [(2, 1), (2, 2)]
+        open: "{" [(2, 0), (2, 1)]
+        open: (ERROR [(2, 1), (2, 2)]
```

Comment:
This will resolve itself when we update tree-sitter to 0.22.6 in {treesitter} due to https://github.com/tree-sitter/tree-sitter/pull/3330. ERROR nodes are treated as extras when they are generated during error recovery:

https://github.com/tree-sitter/tree-sitter/blob/3da7deedd8e0a3decd2db32b76ea51513876914e/lib/src/subtree.c#L547-L559

https://github.com/tree-sitter/tree-sitter/blob/3da7deedd8e0a3decd2db32b76ea51513876914e/lib/src/parser.c#L1220